data/brands/shop/grocery.json

```diff
@@ -4,6 +4,18 @@
     "exclude": {"generic": ["^grocery$"]}
   },
   "items": [
+    {
+      "displayName": "DMart",
+      "id": "dmart-9c65cc",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "brand": "Avenue Supermarts Limited",
```

Comment:
I changed this to say "DMart" in 0b70d2ccc - we usually use the common name here (how people refer to the business) and not the legal name.

Reply:
Thanks! Will keep that in mind.